package.json

```diff
@@ -6,13 +6,14 @@
     "url": "https://github.com/resin-io/text2speech.git"
   },
   "scripts": {
-    "preinstall": "bash deps.sh"
+    "preinstall": "",
```

Comment:
Can probably elide this line altogether (yes it's a fancy word I learned a while back :P)